release-info.yaml

```diff
@@ -0,0 +1,8 @@
+# Add more attributes w.r.t ceph cluster version under specific version.
```

Comment:
@sunilkumarn417 the word `cluster` is not right here. these are ceph version names.

Reply:
updated code as per the suggestions, please review.